src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -45,6 +46,7 @@ public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStra
      * @param mergeTarget remote branch to be merged into current branch
      * @param mergeStrategy merge strategy
      * @param fastForwardMode fast forward mode
+     *
```

Comment:
Whitespace?, this ctor also doesn't seem to set the timeout variable to any default? Is that behaviour ok?

Reply:
In order to be completely assured of its behavior the default value is 10

Reply:
Please don't make changes that are purely whitespace.  Code formatting in the git plugin is messy and inconsistent but is intentionally kept that way so that the fewest conflicts will be generated with the [many pull requests](https://github.com/jenkinsci/git-plugin/pulls?q=is%3Apr+is%3Aclosed+milestone%3ALater) that are pending and closed because the maintainers aren't able to review them.

Reply:
Got it :+1: